Cargo.toml

```diff
@@ -54,7 +54,7 @@ base64-serde = "0.7.0"
 bytes = "1"
 celestia-tendermint = "0.32.1"
 celestia-types = "0.1.1"
-clap = "4"
+clap = "4.5.4"
```

Comment:
Agree with this change. We should probably provide full versions on all crates we are using.